tests/net/socket/getaddrinfo/prj.conf

```diff
@@ -1,5 +1,6 @@
 # General config
 CONFIG_NEWLIB_LIBC=y
+CONFIG_NEWLIB_LIBC_NANO=y
```

Comment:
Any particular reason to enable nano here?

Reply:
It can run out of memory on some platforms with the newlib full variant (see the commit message).

Hopefully, we can replace this with picolibc soon.

Reply:
Sounds good.